console/src/assets/i18n/ja.json

```
@@ -0,0 +1,2095 @@
+      "TITLE": "ロール",
+      "DESCRIPTION": "プロジェクトグラント時に使用するロールを定義します。",
+      "NAME": "名前",
+      "DISPLAY_NAME": "表示名",
+      "GROUP": "グループ",
+      "ACTIONS": "アクション",
+      "ADDTITLE": "ロールの作成",
+      "ADDDESCRIPTION": "新しいロールの情報を入力します。",
+      "EDITTITLE": "ロールの編集",
+      "EDITDESCRIPTION": "ロールの新しい情報を入力します。",
+      "DELETE": "ロールを削除する",
+      "CREATIONDATE": "作成日",
+      "CHANGEDATE": "最終更新日",
+      "SELECTGROUPTOOLTIP": "グループ{{group}}のすべてのロールを選択します。",
+      "OPTIONS": "オプション",
+      "ASSERTION": "認証時にロールを表明する",
+      "ASSERTION_DESCRIPTION": "ロール情報は、Userinfoエンドポイントから送信され、トークンやその他のタイプのアプリケーション設定に応じて送信されます。",
+      "CHECK": "認証時に認可を確認する",
+      "CHECK_DESCRIPTION": "設定されている場合、ユーザーはアカウントにロールが割り当てられている場合にのみ認証を許可されます。",
+      "DIALOG": {
+        "DELETE_TITLE": "ロールの削除",
+        "DELETE_DESCRIPTION": "プロジェクトのロールを削除しようとしています。本当によろしいですか？"
+      }
+    },
+    "HAS_PROJECT": "認証時にプロジェクトを確認する",
+    "HAS_PROJECT_DESCRIPTION": "ユーザーの組織がこのプロジェクトを持っているかどうかを確認します。そうでない場合、ユーザーを認証することはできません。",
+    "TABLE": {
+      "TOTAL": "エントリ合計：",
+      "SELECTION": "選択された要素",
+      "DEACTIVATE": "プロジェクトの非アクティブ化",
+      "ACTIVATE": "プロジェクトのアクティブ化",
+      "DELETE": "プロジェクトの削除",
+      "ORGNAME": "組織名",
+      "ORGDOMAIN": "組織ドメイン",
+      "STATE": "ステータス",
+      "TYPE": "タイプ",
+      "CREATIONDATE": "作成日",
+      "CHANGEDATE": "最終更新日",
+      "RESOURCEOWNER": "所有者",
+      "SHOWTABLE": "テーブルを表示する",
+      "SHOWGRID": "グリッドを表示する",
+      "EMPTY": "プロジェクトは見つかりません"
+    },
+    "TOAST": {
+      "MEMBERREMOVED": "マネージャーが削除されました。",
+      "MEMBERSADDED": "マネージャーが追加されました。",
+      "MEMBERADDED": "マネージャーが追加されました。",
+      "MEMBERCHANGED": "マネージャーが変更されました。",
+      "ROLESCREATED": "ロールが作成されました。",
+      "ROLEREMOVED": "ロールが削除されました。",
+      "ROLECHANGED": "ロールが変更されました。",
+      "REACTIVATED": "アクティブになりました。",
+      "DEACTIVATED": "非アクティブになりました。",
+      "CREATED": "プロジェクトが作成されました。",
+      "UPDATED": "プロジェクトが変更されました。",
+      "GRANTUPDATED": "グラントが変更されました。",
+      "DELETED": "プロジェクトが削除されました。"
+    }
+  },
+  "ROLES": {
+    "DIALOG": {
+      "DELETE_TITLE": "ロールの削除",
+      "DELETE_DESCRIPTION": "ロールを削除しようとしています。本当によろしいですか？"
+    }
+  },
+  "NEXTSTEPS": {
+    "TITLE": "次のステップ"
+  },
+  "IDP": {
+    "LIST": {
+      "TITLE": "IDプロバイダー",
+      "DESCRIPTION": "IDプロバイダーの構成を管理し、ログイン設定で有効にすることができます。",
+      "ACTIVETITLE": "アクティブなIDプロバイダー"
+    },
+    "CREATE": {
+      "TITLE": "プロバイダーの追加",
+      "DESCRIPTION": "以下のプロバイダーから1つ以上選択してください。",
+      "STEPPERTITLE": "プロバイダーの作成",
+      "OIDC": {
+        "TITLE": "OIDCプロバイダー",
+        "DESCRIPTION": "OIDCプロバイダーに必要なデータを入力してください。"
+      },
+      "OAUTH": {
+        "TITLE": "OAuthプロバイダー",
+        "DESCRIPTION": "OAuthプロバイダーに必要なデータを入力してください。"
+      },
+      "JWT": {
+        "TITLE": "JWTプロバイダー",
+        "DESCRIPTION": "JWTプロバイダーに必要なデータを入力してください。"
+      },
+      "GOOGLE": {
+        "TITLE": "Googleプロバイダー",
+        "DESCRIPTION": "Google Identity Providerのクレデンシャルを入力してください。"
+      },
```

```suggestion
      },
      "GITLAB": {
        "TITLE": "Gitlab Provider",
        "DESCRIPTION": "Enter the credentials for your Gitlab Identity Provider"
      },
      "GITLABSELFHOSTED": {
        "TITLE": "Gitlab Self Hosted Provider",
        "DESCRIPTION": "Enter the credentials for your Gitlab Self Hosted Identity Provider"
      },
```

can you please provide the japanese translations? thx